app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,5 @@
+class MoviesController < ApplicationController
+  def new
+    @movie = Movie.new
+  end

```

Comment:
必要なものはnewではなくindexです。モデルがないので，中身は不要です。

```suggestion
  def index
  end
```

Reply:
ご指摘いただいた点を修正しました。ご確認お願いいたします。